src/Sandbox.php

```diff
@@ -237,6 +243,17 @@ public function getActive(): ?bool
         return $this->active;
     }
 
+    public function setShared(bool $shared): self
+    {
+        $this->shared = $shared;
+        return $this;
+    }
+
+    public function getShared(): ?bool
```

Comment:
is it ever possible for it to be null?

Reply:
Right, it's not. And the same for `active` flag. 👍 Fixed https://github.com/keboola/sandboxes-api-php-client/pull/21/commits/a667fdcc48280f40dd4e3f39ec269ca4c2ae6ab0